gemfire/spring-cloud-starter-stream-source-gemfire/src/main/resources/gemfire-source.yml

```diff
@@ -1,4 +1,6 @@
 #
 # Gemfire client pool must be configured with subscriptions enabled
 #
-subscriptionEnabled:true
+gemfire:
+    pool:
+        subscriptionEnabled:true
```

Comment:
We can't use `yml` for such a cases: http://docs.spring.io/spring-boot/docs/1.4.0.BUILD-SNAPSHOT/reference/html/boot-features-external-config.html#boot-features-external-config-yaml-shortcomings.

Not sure if we can overcome it as default somehow from the code, but at least right now we have to convert it into `.properties`.